orc8r/tools/helm/package_lf.sh

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env bash
```

Comment:
This is a complete ~240 lines of duplication for just 3 lines change. This really needs to be abstracted.

Reply:
The plan was just to delete the old file when the license expires. I can rework the code but it was just a way to get it fast